saleor/settings.py

```diff
@@ -175,6 +175,14 @@ def get_bool_from_env(name, default_value):
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = os.environ.get("SECRET_KEY")
 
+PASSWORD_HASHERS = [
+    "django.contrib.auth.hashers.PBKDF2PasswordHasher",
+    "django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher",
+    "django.contrib.auth.hashers.Argon2PasswordHasher",
+    "django.contrib.auth.hashers.BCryptSHA256PasswordHasher",
+    "django.contrib.auth.hashers.BCryptPasswordHasher",
+]
```

Comment:
Maybe we could import the default hasher list from django and add the ones we want instead so we always have the preferred hashers of Django.
Someday `PBKDF2PasswordHasher` may no longer be the default/preferred.

So in the case of this PR it would only add `BCryptPasswordHasher`.



https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/conf/global_settings.py#L535-L541

Reply:
@NyanKiyoshi can you change it if you think it's blocking the merge?